### PR TITLE
Lesson16-merve2

### PR DESCRIPTION
**Pull request by mervebngl**

### CODE REVIEW

File: Lesson16/merve2/script.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+checkAgeCategory(70);
+checkAgeCategory(-5);
+console.log('------------------------');
+// 10. Write a function that takes age as a number and whether they are a student (yes or no).
+// If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote".
+// If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". 
+//If the age is greater than 65, print "You are a senior citizen". 
+//If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+
+function checkVotingEligibility(age, isStudent) {
```
Comment (annasvst) [inline Lesson16/merve2/script.js:121]: For values like 'yes'/'no', in programming it's common to use Boolean type value. So change function body like this:

```
if (age < 18 && isStudent) {
        console.log("You are a student and not old enough to vote.");
    } else if (age >= 18 && age <= 65 && !isStudent) {
        console.log("You are eligible to vote.");
    } else if (age > 65) {
        console.log("You are a senior citizen.");
    } else if (isStudent && age >= 18) {
        console.log("You are old enough to vote.");
    }
```


See more info about booleans here: https://www.classes.cs.uchicago.edu/archive/2021/spring/11111-1/happycoding/p5js/if-statements.html#:~:text=An%20if%20statement%20checks%20a,when%20that%20value%20is%20true%20.

When calling the function, you need to call it like this: `voterage(45,true);`

**Review by annasvst — CHANGES_REQUESTED** [review]
Looks good overall, I left a comment about one exercise that needs correction.

File: Lesson16/merve2/script.js (hunk 1 above)
Comment (mervebngl) [reply Lesson16/merve2/script.js:121]: I thought about the same mistake but I didn't know how to find a solution. Thanks :)

**Review by annasvst — APPROVED** [review]
Looks good, approved ✅

**Pull request merged**